lib/linguist/languages.yml

```diff
@@ -4622,6 +4622,14 @@ P4:
   tm_scope: source.p4
   ace_mode: text
   language_id: 348895984
+PDDL:
+  type: programming
+  color: "#0d00ff"
+  extensions:
+  - ".pddl"
+  tm_scope: none
```

Comment:
You're going to need a scope for syntax highlighting. This will be the scope in the grammar.

Reply:
```suggestion
  tm_scope: source.pddl
```